workers/unity/Assets/Fps/Tests/EditmodeTests/ChosenDeploymentAlphaLocatorFlowTests.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using Fps.Connection;
+using Improbable.Gdk.TestUtils.Editor;
+using Improbable.Gdk.Tools;
+using Improbable.Worker.CInterop;
+using NUnit.Framework;
+
+namespace Fps.EditmodeTests
+{
+    [TestFixture]
+    public class ChosenDeploymentAlphaLocatorFlowTests
+    {
+        private SpatialdManager spatiald;
+        private Improbable.Worker.CInterop.Connection connection;
+
+        [OneTimeSetUp]
+        public void OneTimeSetup()
+        {
+            if (!DevAuthTokenUtils.TryGenerate())
+            {
+                throw new ArgumentException("Could not generate a dev auth token.");
```

Comment:
Is this an actual argument exception?

Reply:
Nah probably not, can be just an `Exception`